thirdeye/thirdeye-pinot/src/main/resources/assets/css/thirdeye.css

```diff
@@ -91,6 +97,23 @@ body {
     margin-bottom: 0px;
     border: 0px;
 }
+
+.navbar-inverse .navbar-brand {
```

Comment:
This is all throw away but I would suggest creating a new rule with a higher specificity as opposed to overwriting bootstrap classes.
Maybe a new class, in a css file that gets loaded in after bootstrap